docs/changes.rst

```diff
@@ -42,6 +42,13 @@ Released: not yet
 * Added a 'pywbemlistener' command for running and managing WBEM listeners.
   (issues #430, #479, #948)
 
+* Implement server schema command that returns information about the schema
```

Comment:
I suggest using plural for "about the schema", since there may be multiple schemas in a namespace.

Reply:
DONE